backend/src/test/kotlin/org/genspectrum/dashboardsbackend/controller/SubscriptionsControllerTriggerEvaluationTest.kt

```diff
@@ -0,0 +1,107 @@
+package org.genspectrum.dashboardsbackend.controller
+
+import com.ninjasquad.springmockk.MockkBean
+import io.mockk.every
+import io.mockk.mockk
+import org.genspectrum.dashboardsbackend.api.Organism
+import org.genspectrum.dashboardsbackend.model.triggerevaluation.AggregatedData
+import org.genspectrum.dashboardsbackend.model.triggerevaluation.LapisAggregatedResponse
+import org.genspectrum.dashboardsbackend.model.triggerevaluation.LapisClient
+import org.genspectrum.dashboardsbackend.model.triggerevaluation.LapisClientProvider
+import org.genspectrum.dashboardsbackend.model.triggerevaluation.LapisError
+import org.genspectrum.dashboardsbackend.model.triggerevaluation.LapisInfo
+import org.genspectrum.dashboardsbackend.model.triggerevaluation.LapisResponse
+import org.junit.jupiter.api.Test
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc
+import org.springframework.boot.test.context.SpringBootTest
+import org.springframework.context.annotation.Import
+import org.springframework.http.HttpStatusCode
+import org.springframework.http.MediaType
+import org.springframework.http.ProblemDetail
+import org.springframework.test.web.servlet.result.MockMvcResultMatchers.content
+import org.springframework.test.web.servlet.result.MockMvcResultMatchers.jsonPath
+import org.springframework.test.web.servlet.result.MockMvcResultMatchers.status
+
+@SpringBootTest
+@AutoConfigureMockMvc
+@Import(SubscriptionsClient::class)
+class SubscriptionsControllerTriggerEvaluationTest(
+    @Autowired private val subscriptionsClient: SubscriptionsClient,
+) {
+    @MockkBean
+    private lateinit var lapisClientProviderMock: LapisClientProvider
+
+    @Test
+    fun `GIVEN lapis returns count greater than threshold WHEN evaluating trigger THEN returns condition met`() {
+        val userId = getNewUserId()
+
+        val createdSubscription = subscriptionsClient.postSubscription(dummySubscriptionRequest, userId)
```

Comment:
No serious complaint, but can we somehow overwrite the value of the trigger thershold here from the dummySubscriptionRequest through a copy, or something similar? Only from reading the test it is not clear, why this test should pass and the dummyRequest lies in another file.